src/comm/LinkManager.cc

```diff
@@ -620,7 +622,6 @@ void LinkManager::_updateAutoConnectLinks(void)
             }
         }
     }
-
 #ifndef __android__
```

Comment:
@DonLakeFlyer Is there a way around this "android only supports a single serial connection"?  Having a SiK Radio plugged in via FTDI and the RTK GPS results in QGC only detecting only one or the other.

Reply:
Possibly but I'm not quite sure what it would take to make that work.

Reply:
How does QGC differentiate between connected serial devices? It looks like it does it by the vendorID and productID, in which case most FTDIs are the same ("vendorID": 1027, "productID": 24577)

See this https://github.com/mavlink/qgroundcontrol/blob/master/src/comm/USBBoardInfo.json#L36